src/mock-doc/element.ts

```diff
@@ -267,6 +267,27 @@ export class MockStyleElement extends MockHTMLElement {
   }
 }
 
+//Based on deprecated SVGMatrix - http://html5index.org/SVG%20-%20SVGMatrix.html
+export const defaultSVGMatrix = {
+  a: 1,
```

Comment:
For my own understanding, why do `a` and `d` get assigned a value of 1, but the other single letter properties of the matrix are assigned a value of 0?